Task-WebRTC.md

```diff
@@ -374,15 +374,19 @@ an `Array`. Elements of this array SHALL either be `Nil` to indicate the
 end of candidates towards WebRTC or a `Map` containing the following
 fields:
 
-* The *candidate* field SHALL contain an SDP `candidate-attribute` as
-  defined in the WebRTC specification in string representation.
+* The *candidate* field SHALL contain an SDP `candidate-attribute` or an
+  empty string as defined in the WebRTC specification in string
+  representation.
```

Comment:
"in the WebRTC specification in string representation" sounds a bit weird. Why not just "defined in the WebRTC specification"?